templates/about_us/index.html

```diff
@@ -41,6 +41,11 @@ <h3 style="padding-top: 50px; margin-top: -40px;" id="lib">Libraries</h3>
 
         </ul>
 
+        <h3 style="padding-top: 50px; margin-top: -40px;" id="lib">How to Cite GraphSpace</h3>
+            <a href="https://academic.oup.com/bioinformatics/article/33/19/3134/3867143">GraphSpace: Stimulating Interdisciplinary Collaborations in Network Biology, </a>
```

Comment:
Please add the following sentence before the citation to the paper: "If you use GraphSpace in your research, please cite the following paper."

Reply:
done.